filter_plugins/oo_filters.py

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/python
 # -*- coding: utf-8 -*-
 # vim: expandtab:tabstop=4:shiftwidth=4
+# pylint: disable=too-many-lines
```

Comment:
This is indeed a problem, but not one that you introduced.

/cc @sdodson can you flag the fact this is required as tech debt in the backlog?